### ISSUE TITLE
Fine-tuning procedure for mb_melgan vocoder, Voice Quality degrading with Fine-tuning.

### ISSUE THREAD

**Issue by WadoodAbdul** [TensorSpeech/TensorFlowTTS#608]
Hello!
I'm trying to finetune a Vocoder model for Indian accents.
I've followed the suggestions from [thread #296](https://github.com/TensorSpeech/TensorFlowTTS/issues/296#issuecomment-741850534) and have arrived at a suitable acoustics model.

To improve the output voice quality of the present vocoder( multiband_melgan.v1) model, I had followed the finetuning process mentioned in [examples/multiband_melgan](https://github.com/TensorSpeech/TensorFlowTTS/tree/master/examples/multiband_melgan) with [940000.h5 multiband_melgan.v1-EN](https://drive.google.com/drive/folders/1ne89z914YHdlZHfcRLOAQHuzpRC3c9h-) as the pretrained model.

However the output has degraded(completely muffled speech) compared to the pretrained vocoder.

I had used the same dataset as the one used for fastspeech model training, with this command, 
```
python ./examples/multiband_melgan/train_multiband_melgan.py \
--train-dir ./dump/train/ \
--dev-dir ./dump/valid/ \
--outdir ./examples/multiband_melgan/exp/train.multiband_melgan.v1/ \
--config ./examples/multiband_melgan/conf/multiband_melgan.v1.yaml \
--use-norm 1 \
--pretrained mb_melgan_generator.h5 
```
These are the loss plots that I obtained,
![eval](https://user-images.githubusercontent.com/52132963/124466061-d35e7700-ddb3-11eb-9112-120c9383c34e.png)
![train](https://user-images.githubusercontent.com/52132963/124466070-d8bbc180-ddb3-11eb-9ecd-8337cfc08eaa.png)

Please help me debug this problem, 
thank you

**Comment by dathudeptrai** [comment]
@WadoodAbdul did you continue training after 200k steps ?, even you used pretrained, you still need training around 1M steps :D.

**Comment by WadoodAbdul** [comment]
Thanks for the help, @dathudeptrai. I Had not continued after 200k steps.

Just for clarification, I have to tune the Generator+Discriminator for the rest of the steps(1M), not just the generator for 1M steps. Is that correct?

**Comment by dathudeptrai** [comment]
> Thanks for the help, @dathudeptrai. I Had not continued after 200k steps.
> 
> Just for clarification, I have to tune the Generator+Discriminator for the rest of the steps(1M), not just the generator for 1M steps. Is that correct?

yes :D

**Comment by WadoodAbdul** [comment]
Thanks for the clarification @dathudeptrai :)

**Comment by WadoodAbdul** [comment]
Hey @dathudeptrai ,
the vocoder output at 370k is still not close to human speech. Is this commonly observed and it tunes out better at higher steps?
I'm trying to keep a close watch since this training will span several days on my single GPU

I have attached a comparison of wav in case you want to have a look at it. 
![b'hiMale_engtext_2164'](https://user-images.githubusercontent.com/52132963/125242310-267f7f00-e30a-11eb-9af8-84d9c6fcbd6a.png)
Thanks!
